### PR TITLE
[WIP] Solve issue #226 dubious applications

### LINKED ISSUE #226: Separate review for dubious applications
Find a way to mark dubious applications, review them separately and then get them back to the voting system.

My first approach would be to:
- Have an extra button in the vote view that separates it for an in-depth review. 
- Applications that need in-depth review should not be in the review tab rotation.
- Have a list view of this separated applications.
- View to review a single application and bring it back to voting/reject it.
- Add comments to the application automatically when marked as to be reviewed, and marked as valid or invalid so that further people reviewing it have context around it.

We would need to add an extra state on the application model for this. Everything else should be easy to do.

### PR DESCRIPTION
**Pull request by sanchyalforn**
## What this PR does / Why we need it
Adds a new status on the application model and let you set an application as dubious (to review it later).

Fixes #226 

## Some questions
- [ X ] I have read the contributing guidelines
- [ X ] I abide by this repository Code of Conduct
- [ X ] I understand that my PR won't be merged until Travis gives a "green light"

### CODE REVIEW

File: organizers/templates/invite_list.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
         </button>
         {% if teams %}
             <a href="{% url 'invite_list' %}" class="btn btn-default btn-block">View applications</a>
-        {% elif h_team_enabled %}
+        {% elif h_team_enabled %}s
```
Comment (casassg) [inline organizers/templates/invite_list.html]: This probably doesn't belong here.

File: organizers/views.py [hunk 2]
```diff
@@ -44,6 +44,10 @@ def add_comment(application, user, text):
     return comment
 
 
+def set_dubious(application):
+    application.set_dubious()
```
Comment (casassg) [inline organizers/views.py]: Is this function used?

**Review by casassg — CHANGES_REQUESTED** [review]
lgtm, just a few minor changes before merging.

Also, `contacted` or `assigned` doesn't seem to appear anywhere in the UI. Are those fields used at all? Or is it WIP?

File: organizers/templates/invite_list.html (hunk 1 above)
Comment (sanchyalforn) [reply organizers/templates/invite_list.html]: Sure, not done yet.

File: organizers/views.py (hunk 2 above)
Comment (sanchyalforn) [reply organizers/views.py]: I think this hasn't been written by me, I call application object directly

File: organizers/templates/invite_list.html (hunk 1 above)
Comment (casassg) [reply organizers/templates/invite_list.html]: remember to remove this first

Comment (casassg) [reply organizers/templates/invite_list.html]:
```suggestion
        {% elif h_team_enabled %}
```

**Review by casassg — APPROVED** [review]
please remove that "s" there before merging. It has no purpose XD

everything else lgtm

**Pull request merged**